.coderabbit.yaml

```
@@ -0,0 +1,52 @@
+  changed_files_summary: true    # 변경된 파일 요약
+
+
+  path_filters:
+    - '!node_modules/**'
+    - '!dist/**'
+    - '!build/**'
+    - '!coverage/**'
+    - '!*.lock'
+
+
+
```

YAMLlint (1.37.1)

[error] too many blank lines (3 > 2)

(empty-lines)